lib/web/apiserver.go

```diff
@@ -2314,6 +2314,16 @@ func (h *Handler) siteNodeConnect(
 		}
 	}
 
+	// If the participantMode is not specified, and the user is the one who created the session,
+	// they should be in Peer mode. If not, default to Observer mode.
```

Comment:
After a Slack discussion, it was decided that this would not be a worthwhile test since we would only be able to verify the configuration of the generated terminal, which is being set just a few lines below this.